providers/infrastructure-aws/v2.0.2/cconly/overlay-kube-apiserver-admission.yaml

```
@@ -0,0 +1,192 @@
+            enum: ["", "privileged", "baseline", "restricted"]
+            description: "audit sets the level for the audit PodSecurityConfiguration mode. One of \"\", privileged, baseline, restricted."
+            nullable: true
+          auditVersion:
+            type: string
+            #! Version defaults to v1.24 for now. When v1.25 is part of tanzu-framework, the defaulting should be done
+            #! by a defaulting webhook instead for extended logic.
+            default: "v1.24"
+            description: "auditVersion sets the version for the audit PodSecurityConfiguration mode."
+          warn:
+            type: string
+            enum: ["", "privileged", "baseline", "restricted"]
+            description: "warn sets the level for the warn PodSecurityConfiguration mode. One of \"\", privileged, baseline, restricted."
+            nullable: true
+          warnVersion:
+            type: string
+            #! Version defaults to v1.24 for now. When v1.25 is part of tanzu-framework, the defaulting should be done
+            #! by a defaulting webhook instead for extended logic.
+            default: "v1.24"
+            description: "warnVersion sets the version for the warn PodSecurityConfiguration mode."
+          exemptions:
+            type: object
+            description: "exemption configuration for the PodSecurityConfiguration."
+            properties:
+              namespaces:
+                type: array
+                items:
+                  type: string
+                description: "namespaces excluded to apply PodSecurityConfiguration Admission."
+  #@overlay/remove
+  #@overlay/match by=overlay.map_key("name"),missing_ok=True
+  - name: eventRateLimitConf
+  #@overlay/append
+  - name: eventRateLimitConf
+    required: false
+    schema:
+      openAPIV3Schema:
+        type: string
+        default: ""
+  #@overlay/match missing_ok=True
+  patches:
+  #@overlay/remove
+  #@overlay/match by=overlay.map_key("name"),missing_ok=True
+  - name: admissionConfiguration
+  #@overlay/append
+  - name: admissionConfiguration
+    description: "Adds an admission configuration for kube-apiserver."
+    definitions:
+    - selector:
+        apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+        kind: KubeadmControlPlaneTemplate
+        matchResources:
+          controlPlane: true
+      jsonPatches:
+      - op: add
+        path: "/spec/template/spec/kubeadmConfigSpec/clusterConfiguration/apiServer/extraArgs/admission-control-config-file"
+        value: "/etc/kubernetes/admission-control-config.yaml"
+      - op: add
+        path: "/spec/template/spec/kubeadmConfigSpec/clusterConfiguration/apiServer/extraVolumes/-"
+        valueFrom:
+          template: |
+            - name: admin-control-conf
+              hostPath: /etc/kubernetes/admission-control-config.yaml
+              mountPath: /etc/kubernetes/admission-control-config.yaml
+              readOnly: true
+              pathType: "File"
```

```suggestion
            name: admin-control-conf
            hostPath: /etc/kubernetes/admission-control-config.yaml
            mountPath: /etc/kubernetes/admission-control-config.yaml
            readOnly: true
            pathType: "File"
```
This needs to be a dict because we append to the array, not an array